tests/conftest.py

```diff
@@ -56,8 +38,30 @@ def tempdir(tmp_path_factory):
 
 @pytest.fixture(scope="session")
 def http_server_url(tempdir, request):
```

Comment:
```suggestion
def http_server_url(tempdir):
```
Looks like `request` is no longer used here.